pkg/blockstorage/awsefs/awsefs.go

```diff
@@ -432,9 +430,10 @@ func (e *Efs) SnapshotCreate(ctx context.Context, volume blockstorage.Volume, ta
 	}, nil
 }
 
-func (e *Efs) createK10DefaultBackupVault() error {
+// Create a Backup Vault, also checks if vault already exist
+func (e *Efs) CreateDefaultBackupVault() error {
```

Comment:
We should not include "Default" in the name anymore. Your PR now makes it mandatory for the consumer to pass in a vault name. So that means there is no notion of a default vault name anymore. Rename it to CreateBackupVault ?

Reply:
Changed to `CreateBackupVaultWrapper()` since the Backup API is `CreateBackupVault()`